### ISSUE TITLE
zip file damaged

### ISSUE THREAD

**Issue by d4op** [Chocobozzz/OpenVPN-Admin#117]
all works fine.
when i download the certs via zip.
the zip is damaged why ?
permission is right and files works when i download manaually from webroot folder via sftp.

**Comment by d4op** [comment]
php zip module installed

**Comment by faglo** [comment]
`sudo service apache2 restart`

**Comment by d4op** [comment]
Didnt work

Von meinem iPad gesendet

Am 03.08.2018 um 17:24 schrieb faglo <notifications@github.com<mailto:notifications@github.com>>:


sudo service apache2 restart

**Comment by SilverSnake83** [comment]
Yes I have the same issue the zip is corrupted once downloaded. 
I'm running on windows, winrar and the explorer can't open it

**Comment by lagman** [comment]
Can you check if its zip file, and not a text file with .zip extension.
I had this issue on CentOS 7 due to selinux not allowing to run the zip command on client directory.
check the logs, enable more verbose logs on your webserver, check selinux logs also.
and also the permission of openvpn-admin directory and sub-directories.

**Comment by SilverSnake83** [comment]
Indeed the zip file is in fact a txt file but it looks more like a .html file.
[openvpn-windows.txt](https://github.com/Chocobozzz/OpenVPN-Admin/files/2500412/openvpn-windows.txt)

**Comment by lagman** [comment]
Take a look at the webserver log, you either don't have permission to write to "client-conf/" directory, or selinux is disallowing access, if you have it installed. Its on by default on Centos7.
selinux wont allow the zip scripts and the sql connect script to run by default.
The logs should show you where the scripts are failling.

**Comment by hmidani-abdelilah** [comment]
just change owner and  permission 
`chown www-data:www-data -R /var/www/`
`chmod 755 -R /var/www/`
